code/lib/preview-api/src/modules/client-api/ClientApi.ts

```diff
@@ -55,61 +51,61 @@ const checkMethod = (method: keyof typeof warningAlternatives) => {
     );
   }
 
-  if (!singleton) {
+  if (!global.__STORYBOOK_CLIENT_API__) {
```

Comment:
This is code that runs in a browser, right?  If so, should this not be `globalThis`?  AFAIK, `global` is only valid in node.js.

Reply:
`global` is imported from a module that does some switcharoo with multiple environments. For all intends and purposes it's the global scope in each environment

Reply:
I have this PR:
https://github.com/storybookjs/storybook/pull/20184/files

That will actually make all this a lot better, but that's not something I'm ready to merge as part of 7.0..